handler/oauth2/flow_authorize_code_token.go

```diff
@@ -5,197 +5,99 @@ package oauth2
 
```

Comment:
This diff is quite challenging to read, would it be possible to reduce the refactoring and separating into different files? Since this is security relevant code, the smaller the diff the faster the review :)

Reply:
We will try to revert the refactored code and push the changes in the coming days